src/MISC/fix_ipi.cpp

```diff
@@ -322,7 +322,8 @@ void FixIPI::initial_integrate(int /*vflag*/)
   double *boxhi = domain->boxhi;
   double *boxlo = domain->boxlo;
   double posconv;
-  posconv=0.52917721*force->angstrom;
+  if (force->boltz == 1.0) posconv = 1.0; // if one uses LJ units, posconv should be 1
```

Comment:
To check for the current units setting, you should do a string compare of `update->unit_style`:

```suggestion
  if (strcmp(update->unit_style,"lj") == 0) posconv = 1.0; // if one uses LJ units, posconv should be 1
```